### PR TITLE
GAWB-2758: capture ontology node info

### PR DESCRIPTION
**Pull request by ahaessly**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/library/attribute-definitions.json [hunk 1]
```diff
@@ -240,59 +240,77 @@
...
+    "library:DS_URL": {
+      "type": "array",
+      "items": { "type": "string" },
+      "hidden": true
     },
     "library:NCU": {
       "title": "Future commercial use is prohibited",
       "type": "boolean",
       "renderHint": {"wording": "yes/no"},
+      "indexable": false,
```
Comment (rushtong) [inline src/main/resources/library/attribute-definitions.json:283]: Do we need to specify indexable false here? See https://github.com/broadinstitute/firecloud-orchestration/blob/3df70194b3ac98b3134ac33a6cc810784e3884a2/src/main/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSupport.scala#L88

On that line, we remove the indexable fields from the document that we've already used for generating anything data-use related. That includes this one, and several others that are also annotated below.

Comment (davidangb) [reply src/main/resources/library/attribute-definitions.json:283]: Greg, I think you're right that it's not needed, but it also doesn't hurt to have indexable: false. Now it's *really* not indexed.

**Review by rushtong — APPROVED** [review]
👍

**Review by davidangb — APPROVED** [review]
:+1:, assuming the UI PR doesn't require any changes here.

**Pull request merged**